update.py

```diff
@@ -902,56 +942,25 @@ def runPostProcessing():
                 *css_filenames
             )
 
-            if not os.path.exists(output_filename):
+            if not os.path.exists(f"output{output_filename}"):
+                # Simply concatenate CSS files - let PostCSS handle the processing
                 merged_css = "\n".join(
                     getFileContents(css_filename)
                     for css_filename in sorted(css_filenames, key=lambda x: "my_" in x)
                 )
 
-                # Do not display fonts on mobile devices.
-                merged_css = re.sub(
-                    r"@font-face\{(?!.*?awesome)(.*?)\}",
-                    r"@media(min-width:901px){@font-face{\1}}",
-                    merged_css,
-                    flags=re.S,
-                )
-                merged_css = re.sub(
-                    r"@font-face\{([^)]*?Lato)(.*?)\}",
-                    r"",
-                    merged_css,
-                    flags=re.S,
-                )
-                merged_css = merged_css.replace("Lato", "ui-sans-serif")
-                merged_css = re.sub(
-                    r"@font-face\{([^)]*?Roboto Slab)(.*?)\}",
-                    r"",
-                    merged_css,
-                    flags=re.S,
-                )
-                merged_css = merged_css.replace(
-                    "Roboto Slab",
-                    "Rockwell, 'Rockwell Nova','Roboto Slab','DejaVu Serif','Sitka Small',serif",
-                )
-                merged_css = re.sub(
-                    r"@font-face\{(.*?)\}",
-                    r"@font-face{font-display:swap;\1}",
-                    merged_css,
-                    flags=re.S,
-                )
-
-                merged_css = merged_css.replace(
-                    "@media(min-width: 1200px)", "@media(min-width: 1500px)"
-                )
-                merged_css = merged_css.replace(
-                    "@media(min-width: 992px)", "@media(min-width: 1192px)"
-                )
-
-                # Strip comments and trailing whitespace (created by that in part)
-                merged_css = re.sub(r"/\*.*?\*/", "", merged_css, flags=re.S)
-                merged_css = re.sub(r"\s+\n", r"\n", merged_css, flags=re.M)
+                # Process with PostCSS
+                processed_css = processWithPostCSS(merged_css)
 
+                # Validate processed CSS: fallback to original if empty or invalid
+                if not processed_css or not isinstance(processed_css, str) or processed_css.strip() == "":
+                    my_print("PostCSS processing failed or returned empty content, using original CSS.")
+                    css_to_write = merged_css
+                else:
+                    css_to_write = processed_css
                 putTextFileContents(
```

Comment:
Just as a note to self, ought to use changeFileContents function or such, to avoid flicker in case of unchanged contents, although probably it's always change it seems to me.